util/gnatsd.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=NATS messaging server
+After=network.target
+
+[Service]
+PrivateTmp=true
+Type=simple
+PIDFile=/var/run/gnatsd/%i.pid
+ExecStart=/usr/sbin/gnatsd -c /etc/gnatsd.conf
```

Comment:
I'd recommend adding the following commands for standard reload and stop behavior:

```
ExecReload=/bin/kill -s HUP $MAINPID
ExecStop=/bin/kill -s QUIT $MAINPID
```

Reply:
Are you sure the QUIT signal is the right signal to terminate gnatsd correctly? Because gnatsd spams the logs when you send it the QUIT sig: https://pastebin.com/9D4R40b4

Reply:
TERM would probably be better?

Reply:
https://github.com/nats-io/gnatsd/blob/master/server/signal.go#L32

Reply:
other like TERM would not allow server to cleanly shutdown (which may be OK as a standalone process).